core/new-gui/src/environments/environment.default.ts

```diff
@@ -33,8 +33,17 @@ export const defaultEnvironment = {
    */
   downloadExecutionResultEnabled: false,
 
+  /**
+   * whether user system is enable
+   */
   userSystemEnabled: false,
 
+  /**
+   * whether user dashboard is enable
+   * TODO: to combine with `userSystemEnabled`
+   */
+  userDashboardEnabled: false,
```

Comment:
please combine it with `userSystemEnabled` in this PR, there's no need to introduce another env variable